app/views/projects/show.html.erb

```diff
@@ -92,8 +92,8 @@
                 </a>
               <% end %>
               <%= link_to create_fork_project_path(@project), class: "btn primary-button projects-primary-button" do %>
-                <%= image_tag("SVGs/forkProject.svg", alt: "Fork Project") %>
-                <span>Fork</span>
+                <%= image_tag("SVGs/forkProject.svg", alt: "Create Copy") %>
+                <span>Create Copy</span>
```

Comment:
This shouldn't be create copy. It should be fork. Only for the current user, it can be create copy.

Reply:
#1774